packages/core-bridge/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.node",
   "types": "index.d.ts",
   "scripts": {
-    "build-rust": "TEMPORAL_WORKER_FORCE_BUILD=true node ./scripts/build.js",
+    "build-rust": "npx cross-env TEMPORAL_WORKER_FORCE_BUILD=true node ./scripts/build.js",
```

Comment:
I don't think you need `npx` in package scripts, the path should already be set here.

Reply:
At this point I'd avoid using env vars and switch to command line arguments.

Reply:
good idea to switch to command line args. I have seen several env variables in build.js starting with "TEMPORAL_". they should all be switched. I can work on that tomorrow.

Reply:
removed environment variables TEMPORAL_WORKER_FORCE_BUILD and TEMPORAL_WORKER_BUILD_TARGETS with argparse command line options. there is no fallback to the env variables.

I do not know which impact this has for the CI/CD pipeline(s). @bergundy: you have to take over.

```
.> node scripts/build.js -h
usage: build.js [-h] [-v] [-t {x86_64-apple-darwin,aarch64-apple-darwin,x86_64-unknown-linux-gnu,aarch64-unknown-linux-gnu,x86_64-pc-windows-gnu,all}] [-f]

build.js compiles the sdk-core via cargo and then packages it as javascript module

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
  -t {x86_64-apple-darwin,aarch64-apple-darwin,x86_64-unknown-linux-gnu,aarch64-unknown-linux-gnu,x86_64-pc-windows-gnu,all}, --targets {x86_64-apple-darwin,aarch64-apple-darwin,x86_64-unknown-linux-gnu,aarch64-unknown-linux-gnu,x86_64-pc-windows-gnu,all}
                        specifies the targets to build for (formerly env.TEMPORAL_WORKER_BUILD_TARGETS)
  -f, --force           forces a clean rebuild of sdk-core and repackaging (formerly env.TEMPORAL_WORKER_FORCE_BUILD)

Happy compiling!
```